Migrate family of `raw` client commands to GRPC V2. (#211)
Concordium/concordium-client: ChangeLog.md

## Unreleased

- `raw` commands now use the V2 GRPC API exposed by the node. This introduces
  some potentially breaking changes:
    - Commands `raw GetTransactionStatusInBlock`, `raw StartBaker`, `raw StopBaker`,
      `raw JoinNetwork` and ` raw LeaveNetwork` have been removed.
    - Command `raw GetBlockSummary` has been removed, and replaced by the commands
      `raw GetBlockPendingUpdates`, `raw GetBlockSpecialEvents`,
      `raw GetBlockChainParameters` `raw GetBlockFinalizationSummary`. These provide
      a more granular way of accessing to the same data.
    - `raw BanNode` and `raw BanNode` no longer support node IDs, but now rather an
      take just an IP address.
    - `raw DumpStart` takes a parameter specifying the path of the file to write
      dumped packets to, and furthermore supports a flag to specify whether raw
      packets should be written to the file.
    - Output of `raw GetBannedPeers` prints a JSON file containing a list of
      strings each corresponding to a banned IP.

>     - Output of `raw GetPeerUptime` now prints an integer representing the uptime

Author: Should this mention the unit? Before it printed a `Maybe Word64`, hence the change.

Reviewer: Yes, in case someone relies on the output (a simple runner script fx).

## Unreleased

- `raw` commands now use the V2 GRPC API exposed by the node. This introduces
  some potentially breaking changes:
    - Commands `raw GetTransactionStatusInBlock`, `raw StartBaker`, `raw StopBaker`,
      `raw JoinNetwork` and ` raw LeaveNetwork` have been removed.
    - Command `raw GetBlockSummary` has been removed, and replaced by the commands
      `raw GetBlockPendingUpdates`, `raw GetBlockSpecialEvents`,
      `raw GetBlockChainParameters` `raw GetBlockFinalizationSummary`. These provide
      a more granular way of accessing to the same data.

>     - `raw BanNode` and `raw UnbanNode` no longer support node IDs, but now rather an
>       take just an IP address.

Reviewer: Something is slightly off in this sentence. Perhaps just?
```suggestion
    - `raw BanNode` and `raw UnbanNode` no longer support node IDs, now they take just an IP address.
```

    - `raw DumpStart` takes a parameter specifying the path of the file to write
      dumped packets to, and furthermore supports a flag to specify whether raw
      packets should be written to the file.

>     - Output of `raw GetBannedPeers` prints a JSON file containing a list of
>       strings each corresponding to a banned IP.

Reviewer: ```suggestion
    - Output of `raw GetBannedPeers` prints a JSON list containing strings, each corresponding to a banned IP.
```

    - Output of `raw GetPeerUptime` now prints an integer representing the uptime
      of the node.

>     - Slight changes to `raw GetNodeInfo` and `raw GetPeerData` output.

Author: This is due to some fields being elaborated a bit. For instance the baker ID is included when the node is part of a baker or finalization committee.

Reviewer: The fact that you made a comment here, suggests that you should add more details to this point in the changelog ;)
